bookie/views/auth.py

```diff
@@ -14,6 +14,7 @@
 from bookie.models import IntegrityError
 from bookie.models.auth import UserMgr
 from bookie.models.auth import ActivationMgr
+from bookie.models.auth import User
```

Comment:
I know it's not your doing but we've been trying to clean up the imports more into the format like:

    from bookie.models.auth import (
        ActivationMgr,
        User,
        UserMgr,
    )